README.md

```diff
@@ -46,9 +46,13 @@ bevy_fmod = { git = "https://github.com/Salzian/bevy_fmod.git", tag = "v0.1.0" }
 
 ## Examples
 
-FMOD Studio comes with an Examples project. Open it and build the project. Then put the bank files inside the assets folder of this repository.
-You can also configure FMOD Studio to build to a folder you specify.
-Run examples with `cargo run --example`. See the source code of the examples for more details.
+To test the examples of this library, clone the repository. FMOD Studio comes with an Examples project. Open it and
+select `File > Save as...`. Save the project as `<bevy_fmod>\assets\fmod\demo_project.fspro`.
```

Comment:
For this to work with the changes in the other files it needs to be `<bevy_fmod>\assets\fmod\demo_project\demo_project.fspro`.

Reply:
This is somewhat of an oddity of FMOD studio. Try it out yourself, but saving the project as described actually does not save the project file at that location. Instead, FMOD creates a new folder with the project name and inserts the project file in there.

So by just following the guide, the user *should*:tm: end up with a working repository.

Reply:
Huh, I had to try it out again but you're right haha